KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -612,5 +535,61 @@ function check_phantom_arguments(nd, ss, us)
             ssy = ss[2]
         end
     end
+    
     return ssx, ssy, ssz, usx, usy, usz
 end
+
+"""
+    T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(labels, tissue_properties = nothing)
+
+This function returns the default brain tissue properties using a labels identifier Matrix
+# Arguments
+- `labels` : (`::Matrix`) the labels identifier matrix of the phantom
+- `tissue_properties` : (`::Dict`, `=Dict()`) phantom tissue properties in ms and Hz considering the available tissues
+
+# Returns
+- `T1, T2, T2s, ρ, Δw`: (`::Matrix`) matrices of the same size of labels with the tissues properties information
+
+# Examples
+```julia-repl
+julia> T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(labels, tissue_properties)
+
+julia> T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(labels)
+```
+"""
+function default_brain_tissue_properties(labels, tissue_properties = Dict())
+    # Load default tissue properties
+    default_properties = Dict(
+        # T1, T2, T2*, ρ, Δw
+        "CSF" => [2.569, 0.329, 0.058, 1, 0],
+        "GM" => [0.833, 0.083, 0.069, 0.86, 0],
+        "WM" => [0.500, 0.070, 0.061, 0.77, 0],
+        "FAT1" => [0.350, 0.070, 0.058, 1, -3.84], #-220 Hz
+        "MUSCLE" => [0.900, 0.047, 0.030, 1, 0],
+        "SKIN/MUSCLE" => [0.569, 0.329, 0.058, 1, 0],
+        "SKULL" => [0, 0, 0, 0, 0],
+        "VESSELS" => [0, 0, 0, 0, 0],
+        "FAT2" => [0.500, 0.070, 0.061, 0.77, -3.84], #-220 Hz
+        "DURA" => [2.569, 0.329, 0.058, 1, 0],
+        "MARROW" => [0.500, 0.070, 0.061, 0.77, 0])
+    
+    tissue_properties = merge(default_properties, tissue_properties)
+    properties = []
+    for i=1:5
+        temp =
+        (labels .== 23) * tissue_properties["CSF"][i] .+ #CSF
+        (labels .== 46) * tissue_properties["GM"][i] .+ #GM
+        (labels .== 70) * tissue_properties["WM"][i] .+ #WM
+        (labels .== 93) * tissue_properties["FAT1"][i] .+ #FAT1
+        (labels .== 116) * tissue_properties["MUSCLE"][i] .+ #MUSCLE
+        (labels .== 139) * tissue_properties["SKIN/MUSCLE"][i] .+ #SKIN/MUSCLE
+        (labels .== 162) * tissue_properties["SKULL"][i] .+ #SKULL
+        (labels .== 185) * tissue_properties["VESSELS"][i] .+ #VESSELS
+        (labels .== 209) * tissue_properties["FAT2"][i] .+ #FAT2
+        (labels .== 232) * tissue_properties["DURA"][i] .+ #DURA
+        (labels .== 255) * tissue_properties["MARROW"][i] #MARROW
+        push!(properties, temp)
+    end
```

Comment:
Just realized this is iterating for (T1, T2, T2*, ρ, Δw) instead of the tissue properties. This code has many problems:

- for loop is hard-coded to `i = 1:5`, what is 5? what if we want more properties? if we change it to 6 it will error.
- the properties are not preallocated, you `push!` them.
- The order of the properties does not make much sense. T1 and T2 before ρ? 
- The Δw = -3.84 doesn't seem correct. 

Suggestion:
```julia
Nproperties = 5 # ρ, T1, T2, T2*, Δw, explicit ... 5 is the number of properties
# or
Nproperties = size(tissue_properties ... , 2) # Or something like that
properties = zeros(size(...), Nproperties)
tissue_labels = [23, 46, ....]
for (label, tissue) in zip(tissue_labels, tissue_properties)
	for property in 1:Nproperties
		properties[:,:, property] .+= (labels .== label) .* tissue[property]
	end
end
```